.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,88 @@
+name: Bug report
+description: Report a bug in pylint
+labels: [bug, needs triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please make sure that this issue wasn't already requested, or already implemented in the main branch.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What is the bug about ?
+      description:
+        Please provide the code you linted, and configurations files and multiple named
+        file if required
+      value: |
+        Given a file `a.py`:
+
```

Comment:
These lines are not really necessary

Reply:
Well I think it nudges the reporter to provide a reproducible example and I really like having a reproducible example, I think I'll keep that.